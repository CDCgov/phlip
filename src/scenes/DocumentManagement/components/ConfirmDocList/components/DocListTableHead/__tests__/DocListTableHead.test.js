import React from 'react'
import { shallow } from 'enzyme'
import { DocListTableHead } from '../index'

const props = {
  onSelectAll: () => {},
  allSelected: false
}

describe('DocumentManagement - DocList - DocListTableHead', () => {
  test('should render correctly', () => {
    expect(shallow(<DocListTableHead {...props} />)).toMatchSnapshot()
  })
})