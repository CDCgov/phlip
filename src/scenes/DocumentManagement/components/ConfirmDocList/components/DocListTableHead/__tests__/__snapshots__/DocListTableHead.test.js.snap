// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DocumentManagement - DocList - DocListTableHead should render correctly 1`] = `
<TableRow
  key="docTableHeaders"
  style={
    Object {
      "width": "100%",
    }
  }
>
  <TableCell
    id="file-name"
    key="file-name"
    padding="checkbox"
    scope="col"
    style={
      Object {
        "width": "100%",
      }
    }
  >
    File Name
  </TableCell>
  <TableCell
    id="uploaded-by"
    key="uploaded-by"
    padding="checkbox"
    scope="col"
    style={
      Object {
        "width": undefined,
      }
    }
  >
    Uploaded By
  </TableCell>
  <TableCell
    id="uploaded-data"
    key="uploaded-data"
    padding="checkbox"
    scope="col"
    style={
      Object {
        "width": undefined,
      }
    }
  >
    Uploaded Date
  </TableCell>
</TableRow>
`;
